.github/workflows/test.yaml

```
@@ -53,7 +53,7 @@ jobs:
         strategy:
             matrix:
                 os: [ubuntu-latest, macos-latest]   # eventually add `windows-latest`
-                python-version: [3.7, 3.8, 3.9]
+                python-version: [3.8, 3.9, "3.10"]
```

```suggestion
                python-version: [3.8, 3.9]
```

we can't use 3.10 yet in Starkland :(